src/data_generators/single_publisher_design.py

```diff
@@ -33,34 +33,96 @@
 
 PUBLISHER_SIZES = [1000 * 2 ** k for k in range(8)]
 
+# We consider four basic data generating processes:
+#   Poisson, Exponential-Poisson, Gamma-Poisson and Zeta.
+# For each of these processes, we consider distributions
+# having mean 1.5, 3, 5 and 10.
+
 IMPRESSION_GENERATORS = [
+    ## Mean 1.5
+    # Shifted Poisson: Mean = 1.5, Var = 0.5
     GeneratorParameters(
-        "Homogeneous", HomogeneousImpressionGenerator, {"poisson_lambda": 2.0}
+        "Homogeneous", HomogeneousImpressionGenerator, {"poisson_lambda": 0.5}
     ),
+    # Exponential Poisson: Mean = 1.5, Var = 0.75
     GeneratorParameters(
-        "Homogeneous", HomogeneousImpressionGenerator, {"poisson_lambda": 5.0}
+        "Heterogeneous",
+        HeterogeneousImpressionGenerator,
+        {"gamma_shape": 1.0, "gamma_scale": 0.5},
     ),
+    # Gamma Poisson: Mean = 1.5, Var = 6
```

Comment:
Nice!  IIUC, you calculated variance using the conditional variance formula:  Var(freq) = E[Var(X | lambda)] + Var[ E(X | lambda) ] = E(lambda) + Var(lambda) = gamma_shape * gamma_scale + gamma_shape * gamma_scale^2,

while E(freq) = 1 + gamma_shape * gamma_scale.  Is that correct?  Please call out the formula to help readers understand.

Reply:
Perhaps just clarify that it is equivalent to a negative binomial and variance is obtained using negative binomial

Reply:
Done.